save-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt

```diff
@@ -0,0 +1,21 @@
+package org.cqfn.save.agent
+
+import org.cqfn.save.domain.TestResultStatus
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property id
+ * @property agentId
+ * @property status
+ * @property startTime
+ * @property endTime
+ */
+@Serializable
+data class TestExecutionDto(
+    val id: Long,
+    var agentId: Long,
```

Comment:
Why do you need `var`s for these fields? I believe dto could be made immutable?

Reply:
Fixed